app/graphql/types/Input/route_input_type.rb

```diff
@@ -0,0 +1,12 @@
+module Types
+  class Input::RouteInputType < Input::BaseInput
+    description "Attributes for creating or updating a route"
+    argument :userEmail, Integer, required: false
+    argument :startTime, String, required: false
```

Comment:
Didn't you change startTime and endTime to be DateTime objects?

Reply:
They are datetime object in the database, but we want to allow the FE to post them as plain strings.  we can parse them into datetimes when we bring them in.  That will be part of sad path testing.